libcxx/test/benchmarks/ContainerBenchmarks.h

```diff
@@ -48,6 +48,76 @@ void BM_Assignment(benchmark::State& st, Container) {
   }
 }
 
+// Wrap any Iterator into an input iterator
+template <typename Iterator>
+class InputIterator {
```

Comment:
Why don't we simply use `cpp17_input_iterator`?

Reply:
Thanks for pointing out the existence of `cpp17_input_iterator`, which could make my benchmark testing simpler. I will adopt this class.